operating/verify-image-signatures.adoc

```diff
@@ -0,0 +1,36 @@
+[id="verify-image-signatures"]
+= Verifying image signatures
+include::modules/common-attributes.adoc[]
+:context: verify-image-signatures
+
+toc::[]
+
+[role="_abstract"]
+You can use {product-title} ({product-title-short}) to ensure the integrity of the container images in your clusters by verifying image signatures against pre-configured keys.
+
+You can create policies to block unsigned images and images that do not have a verified signature.
+You can enforce the policy by using an admission controller to stop unauthorized deployment creation.
```

Comment:
Additionally, while I like the `Additional resources` section, I have the feeling readers may miss the most important part: _which_ setting they have to ensure is enable to also enable this blocking.
Reading through this without prior knowledge, I could see myself expecting this to be done "by default", but `Contact Image Scanners` is disabled by default. Ideally we make the `Enabling admission controller enforcement` link a bit more prominent here, similar to what we have done for the OpenShift Auth provider.
WDYT?